CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing
+
+We would ❤️ for you to contribute to Appwrite and help make it better! As a contributor, here are the guidelines we would like you to follow:
+
+## Code of Conduct
+
+Help us keep Appwrite open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require appwrite/php-runtimes
+```
+
+```php
+<?php
+
+require_once '../vendor/autoload.php';
+
+use Appwrite\Runtimes\Runtimes;
+
+$runtimes = new Runtime();
+
+// returns all supported cloud function runtimes
+Runtimes::getAll();
+```
+
+## Adding a new Runtime
+
+After the Docker image is created, this must be added to the main class of this library.
+
+References to this must be added to the constructor of `src/Runtimes/Runtimes.php`.
+
+Example:
+
+```php
+$dotnet = new Runtime('dotnet', '.NET');
+$dotnet->addVersion('5.0', 'mcr.microsoft.com/dotnet/runtime:5.0-alpine', 'appwrite/env-dotnet-5.0:1.0.0', [System::X86, System::ARM]);
+$runtimes[] = $dotnet;
+```
+
+## Function Runtimes Checklist
+
+The following checklist aims to ensure that a function runtime gets added successfully
+
+- [ ] Implement the runtime in [open-runtimes/open-runtimes](https://github.com/open-runtimes/open-runtimes) (you can find the tutorial [here](https://github.com/open-runtimes/open-runtimes/blob/main/docs/add-runtime.md))
+  - [ ] Prepare the files for your new runtime
+    - [ ] Dockerfile
+    - [ ] Readme
+  - [ ] Write the runtime
+    - [ ] Initialize a web server
+      - [ ] Set Port 3000
+      - [ ] Bind IP 0.0.0.0
+      - On each POST Request
+        - [ ] Check that the `x-internal-challenge header` matches the `INTERNAL_RUNTIME_KEY` environment variable
+        - [ ] Decode the executor's JSON POST request
+    - [ ] Create Request Class
+      - [ ] Fields
+        - [ ] env
+        - [ ] payload
+        - [ ] headers
+    - [ ] Create Response Class
+      - [ ] Functions
+        - [ ] send(string)
+        - [ ] json(object)
+    - [ ] Execute the function
+      - [ ] Add `try catch` block for error handling
+  - [ ] Write the Dockerfile
+    - [ ] Add the Docker image you want to base your runtime off
+    - [ ] Create the folders you'll use
+    - [ ] Copy your source code and set working directory
+    - [ ] Add execute permissions for any scripts
+      - [ ] `build.sh`
+      - [ ] `start.sh`
+    - [ ] Use `RUN` commands for necessary dependencies (if needed)
+    - [ ] Expose port 3000
+    - Add a `CMD` command for your launch script (i.e. `start.sh`)
```

Comment:
It will always be start.sh.. And always. be build.sh

MAYBE there could be prepare.sh I think we have that with PHP. But I also think it could be part of build tho, so let's not recommend it.

Reply:
This line I added based on the instruction [here](https://github.com/open-runtimes/open-runtimes/blob/a277d1b36ebdfb9b0d4fd81866733f8df21b53d1/docs/add-runtime.md?plain=1#L165)

How exactly is `build.sh` being used here?

Reply:
Oh, never mind. This is all under the step of building `Dockerfile`, so it makes sense as it is.

Reply:
Ah okay 👍